### PR TITLE
Update gnome-weather to 40.1

### PR DESCRIPTION
**Pull request by doraskayo**
Also update `gnome-desktop` while at it.

### CODE REVIEW

File: org.gnome.Weather.json [hunk 1]
```diff
@@ -59,8 +59,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.gnome.org/sources/gnome-weather/40/gnome-weather-40.0.tar.xz",
-                    "sha256": "2a35a73ab2408762181d8650b037205c17ef7bcb8dff3cf0b34af1a2de66aeef"
+                    "url": "https://download.gnome.org/sources/gnome-weather/40/gnome-weather-40.1.tar.xz",
```
Comment (doraskayo) [inline org.gnome.Weather.json:62]: Actually, the tarball hasn't been uploaded yet.

Comment (AsciiWolf) [reply org.gnome.Weather.json:62]: Yeah, there are [some issues](https://gitlab.gnome.org/GNOME/gnome-weather/-/merge_requests/85#note_1246182) with the upload.

Comment (AsciiWolf) [reply org.gnome.Weather.json:62]: The tarball has been uploaded today.

Comment (AsciiWolf) [reply org.gnome.Weather.json:62]: The tarball has been uploaded today.

**Comment by AsciiWolf** [comment]
bot, build

File: org.gnome.Weather.json (hunk 1 above)
Comment (doraskayo) [reply org.gnome.Weather.json:62]: We'll need to update the hash of the tarball first. I will try to do so later today.

Comment (doraskayo) [reply org.gnome.Weather.json:62]: Done.

**Comment by doraskayo** [comment]
Seems to work well.

**Comment by AsciiWolf** [comment]
Yep, works fine for me as well. :-)

**Pull request merged**